javascript-loops/main.js

```diff
@@ -0,0 +1,69 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  while (currentNumber % 2 !== 1 && currentNumber <= 20) {
```

Comment:
The first expression in this condition shouldn't be necessary, since you are already incrementing by 2 in each iteration 🤔